app/audit/models/ConfirmClientDetailsAuditModel.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package audit.models
+
+import play.api.libs.json.{JsObject, JsValue, Json}
+import utils.Utilities.JsonUtil
+
+case class ConfirmClientDetailsAuditModel(clientName: Option[String], nino: Option[String],
```

Comment:
I would suspect that all these fields like: 
* clientName, nino, arn etc mandatory in this context, so if agent manage to login and select client UTR we should have accurate picture of associated client